### ISSUE TITLE
Error at extract!!

### ISSUE THREAD

**Issue by stevenking10** [SL-RU/UnderTranslator#2]
![aaa](https://user-images.githubusercontent.com/9283953/36368274-c48727fa-1556-11e8-8def-8208e9e490fc.jpg)
![bbb](https://user-images.githubusercontent.com/9283953/36368293-d5905d32-1556-11e8-8ee3-2903a950fb0e.jpg)

what should I do??

**Comment by SL-RU** [comment]
Please use that fork: https://github.com/fjay69/UnderTranslator

**Comment by stevenking10** [comment]
I have downloaded "download .exe" but same problem

**Comment by stevenking10** [comment]
@SL-RU I could now extract the files and I'm following your guide: https://github.com/SL-RU/UnderTranslator/blob/master/help/help.md
At this step: Open existed: File->Open choose UndertaleRes folder
I get this error:
![aa](https://user-images.githubusercontent.com/9283953/36400245-038287fa-15d0-11e8-8104-651a5a42a342.jpg)

**Comment by SL-RU** [comment]
What undertale version are you using?

**Comment by stevenking10** [comment]
last version: .v31.08.2017

**Comment by stevenking10** [comment]
@SL-RU can you fix it?

**Comment by SL-RU** [comment]
Ask fjay69 about it - I hadn't work with last Undertale

**Comment by stevenking10** [comment]
@SL-RU I can run the game with the app but can't choose UndertaleRes folder

**Comment by fjay69** [comment]
OK, try these steps. Download my UnderTranslator fork. Download my UndertaleTools and copy into UnderTranslator folder. Run UnderTranslator and go to Tools - Extract. Select Undertale folder, then select folder, where Res will be placed,and... that's it. Should work.

**Comment by stevenking10** [comment]
@fjay69 Sadly, I'm getting same message¨

https://user-images.githubusercontent.com/9283953/36400245-038287fa-15d0-11e8-8104-651a5a42a342.jpg

**Comment by stevenking10** [comment]
@fjay69 When I extract it

![123](https://user-images.githubusercontent.com/9283953/36502526-8a94a8f6-174a-11e8-9ab4-990e4c8b616f.jpg)

![456](https://user-images.githubusercontent.com/9283953/36502534-8e6654a2-174a-11e8-8643-96e264ddf6a0.jpg)

**Comment by fjay69** [comment]
File - Open - Select UndertaleRes folder.

**Comment by stevenking10** [comment]
![6666666](https://user-images.githubusercontent.com/9283953/36527494-3f002cea-17b2-11e8-8e32-c0c0b8de4f94.jpg)
![77777](https://user-images.githubusercontent.com/9283953/36527515-54d752c8-17b2-11e8-8b25-d1324f5bf3be.jpg)

@fjay69 Still same problem :(

**Comment by stevenking10** [comment]
@fjay69 I even tried with an older version of Undertale and still have same problem 
Can you please fix it at last version V. 1.08 ?
I really need to translate the game :)

**Comment by fjay69** [comment]
I have no problems with UnderTranslator. Try replace UndertaleRes into another folder with name with latin symbols only. If this does not help, send me UndertaleRes folder.

**Comment by stevenking10** [comment]
@fjay69 Here is the UndertaleRes folder
https://openload.co/f/3ZVk_m0b_tU/UndertaleRes.zip
And please show me how you open the file.

**Comment by fjay69** [comment]
As I said, File - Open - Select UndertaleRes folder. I tried your files, they opened fine. Has you updated UnderTranslator? Try this: https://yadi.sk/d/rMhaGu9C3SheFa Also try to delete lastpath.txt from UndertaleTranslator folder.

**Comment by stevenking10** [comment]
@fjay69 It works now :)
But I need an advice, how do I know which strings I should translate ?

**Comment by fjay69** [comment]
Well... It's all string which not look like variable names and which you can meet in the game :)

**Comment by SajjadRahim** [comment]
@fjay69 @SL-RU Hi, when i added my new font in folder FONT_new it changed in game but not in the tool?

**Comment by SL-RU** [comment]
@SajjadRahim Are you using last version of UnderTranslator?

**Comment by stevenking10** [comment]
@SL-RU Yeah the same one that @fjay69 uploaded to us https://yadi.sk/d/rMhaGu9C3SheFa

**Comment by fjay69** [comment]
I just checked. This feature should work! Send me your FONT_new folder (at least one gmx, png and patch.txt).

**Comment by SajjadRahim** [comment]
yeah last version, and here my font https://mega.nz/#!yUtB2LBT!DExV0jkFtbEcmUuiL93YQelqBk8WU_vOYo6QxWhlxwg

**Comment by fjay69** [comment]
UnderTranslator will show custom fonts only for "fnt_papyrus.font.gmx", "fnt_comicsans.font.gmx", "fnt_maintext.font.gmx", "fnt_plain.font.gmx".

**Comment by fjay69** [comment]
Well, UnderTranslator can't work correct with arabic fonts. http://i101.fastpic.ru/big/2018/0301/ac/e7424e75156d78daa43c8cfae1f5c8ac.png

**Comment by stevenking10** [comment]
@fjay69 use this text "ﻢﻜﻴﻠﻋ ﻡﻼﺴﻟﺍ"

**Comment by stevenking10** [comment]
@fjay69 Hello,
I made a new fonts and I put them in FONT_new. And when I want to start UnderTranslator.exe. The program doesn't work. It show me nothing, no error.
